scripts/lint-style.py

```diff
@@ -149,6 +152,36 @@ def long_lines_check(lines, path):
             errors += [(ERR_LIN, line_nr, path)]
     return errors
 
+def indent_check(lines, path):
+    errors = []
+    indent_lvl = 0
+    in_prf = 0 # counter for nested proof blocks
+    check_rest_of_block = True # we only check uncomplicated syntax
+    for line_nr, line in enumerate(lines, 1):
+        if in_prf > 0 and check_rest_of_block:
+            lstr = line.lstrip(' ') # strip spaces from beginning of line
+            if lstr[0] == '{' and len(line) - len(lstr) != indent_lvl:
```

Comment:
I think you should only do this check if a comma ends the previous line

Reply:
Good idea. That takes care of things like `simp` configs.

Reply:
Done